src/applications/personalization/profile/components/ProfileHeader.jsx

```diff
@@ -110,11 +123,12 @@ const ProfileHeader = ({
             <img
               src={SERVICE_BADGE_IMAGE_PATHS.get(latestBranchOfService)}
               alt={`${latestBranchOfService} seal`}
-              className="profile-service-badge vads-u-padding-right--3"
+              className="vads-u-padding-right--3"
+              style={{ maxHeight: '75px' }}
```

Comment:
this seems fine. But we could get rid of this now https://github.com/department-of-veterans-affairs/vets-website/blob/11a20d54d362141b8559c6433e87389e15bf92ba/src/applications/personalization/profile/sass/profile.scss#L10-L12